packages/v4/src/content/design-guidelines/components/label/label.md

```diff
@@ -17,9 +17,9 @@ Labels are easily customized to fit different use cases. There are two main type
 
 <img src="./img/filled-labels.png" alt="Filled labels" width="646"/>
 
-Use the filled option to add more visual prominence to a label.
+Use the filled option to add more visual prominence to a label. If your use case requires a mix of clickable and non-clickable labels, consider using unfilled for clickable and filled for non-clickable, or vice versa. Whatever convention you choose, be sure to maintain consistency through your UI.
```

Comment:
Do we need to include "or vice versa" here? It makes this seem like a non-recommendation.

Reply:
Agreed, I've removed that piece.